mathics/builtin/__init__.py

```diff
@@ -202,7 +161,7 @@ def sanity_check(cls, module):
 modules = []
 import_builtins(module_names)
 
-_builtins = []
+_builtins_list = []
```

Comment:
`_builtins` is defined previously as a dictionary.  We should remove the bad coding style of picking a name and using type-morphing variable for whatever happens to be needed. 

Also both `_builtins`  (and `_builtins_list`)  are really bad names because they are generic. Another bad habit that is found in the code that should be removed.

Finally, instead of this global variable getting used in various functions in this module, it should be passed as a parameter.